package/MDAnalysis/topology/PrimitivePDBParser.py

```diff
@@ -167,10 +175,9 @@ def _parsebonds(self, atoms):
 
         bonds = set()
         with openany(self.filename, "r") as f:
-            lines = (line[6:].split() for line in f
-                     if line[:6] == "CONECT")
-            for bond in lines:
-                atom, atoms = int(bond[0]), map(int, bond[1:])
+            lines = (line for line in f if line[:6] == "CONECT")
```

Comment:
I make the method hidden and changed it so that it checks if the syntax of the `CONECT` remark is correct.